Sources/Interaction/Style/InteractorStyleManipulator/index.js

```diff
@@ -288,8 +288,12 @@ function vtkInteractorStyleManipulator(publicAPI, model) {
     );
     if (model.currentManipulator) {
       publicAPI.invokeStartInteractionEvent({ type: 'StartInteractionEvent' });
-      model.currentManipulator.setCenter(model.centerOfRotation);
-      model.currentManipulator.setRotationFactor(model.rotationFactor);
+      if (typeof model.currentManipulator.setCenter === 'function') {
+        model.currentManipulator.setCenter(model.centerOfRotation);
+      }
+      if (typeof model.currentManipulator.setRotationFactor === 'function') {
+        model.currentManipulator.setRotationFactor(model.rotationFactor);
+      }
```

Comment:
@jourdain any better way to do this? Could check that `currentManipulator isinstanceof vtkCameraManipulator`. instead, but requires importing `vtkCameraManipulator`.

Reply:
```
model.currentManipulator.set(publicAPI.get('centerOfRotation', 'rotationFactor'), true);
```

Reply:
You will have to rename center to be centerOfRotation on the various manipulator and properly use it that way.

Reply:
If you don't want to do it. Then do that instead...

```
if (model.currentManipulator.setRotationFactor) {
         model.currentManipulator.setRotationFactor(model.rotationFactor);
 }
```